src/kata6.joinNames.js

```diff
@@ -1,3 +1,14 @@
-const joinNames = (namesObj) => {};
+const joinNames = (namesObj) => {
+	const nameArray = namesObj.map(function (x) {
+		return x.name;
+	});
+	if (nameArray.length === 1) {
+		return nameArray[0];
+	}
+	let lastName = nameArray.pop();
+	let nameString = nameArray.join(", ");
+
+	return `${nameString} & ${lastName}`;
```

Comment:
So here, you should always use the const keyword to declare your variables whenever possible. Also, you need not create the `nameString` variable and instead just use the logic in the return function:
```
  const lastName = nameArray.pop();

  return `${nameArray.join(", ")} & ${lastName}`;
```